<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>898</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>securitext</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>14</y>
     <width>1081</width>
     <height>871</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton1{
	background-color:rgba(85, 98, 112, 255);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#pushButton1:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#pushButton1:hover{
	background-color:rgba(255, 107, 107, 255);
}

QPushButton#pushButton2{
	background-color:rgba(255, 107, 107, 0);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#pushButton2:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(85, 98, 112, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#pushButton2:hover{
	background-color:rgba(85, 98, 112, 255);
}

QPushButton#pushButton3{
	background-color:rgba(255, 107, 107, 0);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
}
QPushButton#pushButton3:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(85, 98, 112, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#pushButton3:hover{
	background-color:rgba(85, 98, 112, 255);
}


QPushButton#pushButton3_2{
	background-color:rgba(85, 98, 112, 155);
	color:rgba(255, 255, 255, 200);
	border-radius:8px;
}
QPushButton#pushButton3_2:pressed{
	padding-left:2px;
	padding-top:2px;
	background-color:rgba(85, 98, 112, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#pushButton3_2:hover{
		color:rgba(85, 98, 112, 155);
		background-color:rgba(255, 255, 255, 200);
}

QPushButton#browse{
	background-color:rgba(85, 98, 112, 0);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
	border:1px solid rgba(0, 0, 0, 0);
	border-color:rgba(46, 82, 101, 50);
}
QPushButton#browse:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#browse:hover{
	background-color:rgba(85, 98, 112, 255);
}

QPushButton#clear2{
	background-color:rgba(85, 98, 112, 0);
	color:rgba(255, 255, 255, 200);
	border-radius:5px;
	border:1px solid rgba(0, 0, 0, 0);
	border-color:rgba(46, 82, 101, 50);
}
QPushButton#clear2:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(255, 107, 107, 255);
	background-position:calc(100% - 10px)center;
}
QPushButton#clear2:hover{
	background-color:rgba(85, 98, 112, 255);
}


</string>
   </property>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>175</y>
      <width>751</width>
      <height>515</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 107, 107, 255), stop:1 rgba(85, 98, 112, 255));
border-radius:10px;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>31</number>
    </property>
    <property name="midLineWidth">
     <number>14</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>518</y>
      <width>341</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mountain</family>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 107, 107, 255);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>220</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(85, 98, 112, 255);
font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>decryption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>626</y>
      <width>240</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>DECRYPT</string>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="autoRepeatInterval">
     <number>100</number>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>187</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic Bold</family>
      <pointsize>35</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(85, 98, 112, 255);
font: 75 35pt &quot;Qatar2022 Arabic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>securitext</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>197</y>
      <width>95</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>â†’ home</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>160</y>
      <width>282</width>
      <height>543</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 20);
border-top-right-radius: 15px;
border-bottom-right-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>160</y>
      <width>280</width>
      <height>542</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 255);
border-top-right-radius: 15px;
border-bottom-right-radius: 55px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources/res.qrc">:/Images/ui1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>63</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>357</y>
      <width>431</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 30);
border:2px solid rgba(0, 0, 0, 0);
border-radius:10px;
border-color:rgba(46, 82, 101, 50);
color:rgba(255, 255, 255, 255);
padding-left:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Insert Key here</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>318</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(226, 226, 226)</string>
    </property>
    <property name="text">
     <string>Browse File...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>197</y>
      <width>95</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>â†’ encryption</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Qatar2022 Arabic</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Qatar2022 Arabic&quot;;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>269</y>
      <width>431</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 30);
border:2px solid rgba(0, 0, 0, 0);
border-radius:10px;
border-color:rgba(46, 82, 101, 50);
color:rgba(255, 255, 255, 255);
padding-left:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Choose File</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>410</y>
      <width>431</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 30);
border:2px solid rgba(0, 0, 0, 0);
border-radius:10px;
border-color:rgba(46, 82, 101, 50);
color:rgba(255, 255, 255, 150);
padding-top:10px;
padding-left:10px</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>#Decrypted message will be displayed here</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::VisualMoveStyle</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="clear2">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>318</y>
      <width>100</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(226, 226, 226)</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <zorder>label_1</zorder>
   <zorder>pushButton1</zorder>
   <zorder>label_6</zorder>
   <zorder>label_5</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton2</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>browse</zorder>
   <zorder>pushButton3</zorder>
   <zorder>pushButton3_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit</zorder>
   <zorder>clear2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
